### ISSUE TITLE
No tests found on Jest 25

### ISSUE THREAD

**Issue by just-jeb** [just-jeb/angular-builders#673]
I'm having this issue with the latest Jest version installed (25.1.0), it was fine when I was using Jest 24.9.0. These are 1 release away from each other [Releases](https://github.com/facebook/jest/releases)

angular-builders/jest version is 8.3.2.

Running:
```
ng test base-lib
```

Results in:
```
  testMatch: /Users/user/Documents/angular-app/projects/base-lib/**/+(*.)+(spec|test).+(ts|js)?(x) - 0 matches
  testPathIgnorePatterns: /node_modules/ - 2796 matches
  testRegex:  - 0 matches
```

_Originally posted by @jonathanlie in https://github.com/just-jeb/angular-builders/issues/615#issuecomment-580065774_

**Comment by just-jeb** [comment]
I had the same issue as above, after updating 24.9.0 to 25.1.0

Seems that **testMatch** should be adjusted as follow: `**/*(*.)@(spec|test).[tj]s?(x)`

p.s. The related **jest** discussion can be found here: [testMatch on Windows](https://github.com/facebook/jest/issues/7914#issuecomment-464352069)

_Originally posted by @it-ias in https://github.com/just-jeb/angular-builders/issues/615#issuecomment-580207632_

**Comment by jonathanlie** [comment]
thank you for the swift response, looking forward to the next release fixing this!

**Comment by just-jeb** [comment]
Should be fixed in `v9.0.0-beta.3`. Please confirm.
